modules/ossm-auto-route.adoc

```diff
@@ -1,6 +1,5 @@
 ////
 This TASK module included in the following assemblies:
-// * service_mesh/v1x/customizing-installation-ossm.adoc
 // * service_mesh/v2x/customizing-installation-ossm.adoc
```

Comment:
Where this page can this be found? I didn't find it in the deploy preview...

Reply:
I moved it from that location but didn't change that comment. Now that I've taken another look, I am moving it to the traffic management section. https://ossmdoc-148--ocpdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-traffic-manage.html#ossm-auto-route_routing-traffic

I'm working on the linking. A few months ago, the link wasn't possible because our doc app didn't support it. It may now. Let me see what I can do.

Reply:
@jwendell Ok, I moved the item and added the link. Does that work for you?

Reply:
@jwendell I see the link in the preview now. It does take some time to refresh and the amount of time varies. I didn't expect you to respond so quickly! :)